main.go

```diff
@@ -337,6 +337,30 @@ func getSummaryOfAgency(c echo.Context) error {
 	return c.JSON(http.StatusOK, agencySummary)
 }
 
+func downloadData(c echo.Context) error {
+	year, err := strconv.Atoi(c.Param("ano"))
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro ano=%d inválido", year))
+	}
+	month, err := strconv.Atoi(c.Param("mes"))
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro mês=%d", month))
+	}
+	agName := c.Param("orgao")
+	agMI, err := client.GetOMA(month, year, agName)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro ano=%d, mês=%d ou nome do orgão=%s são inválidos", year, month, agName))
+	}
+	format := c.QueryParam("format")
```

Comment:
Retornar erro bacana quando "format" estiver vazio

Reply:
Fireman, requestei teu review de novo porque tive uma dúvida aqui. Devo voltar um erro quando o format for vazio ou devo deixar fazer o download default (decidi json)? coloquei pra retornar uma badrequest

Reply:
JSON é um bom default. Penso que, quando o valor definido padrão é bom*, não precisamos obrigar o usuário a setar o parâmetro. É esse um excelente caso para o tal parâmetro opcional.

* nesse caso bom para o usuário e para o sistema, que pode servir sem precisar fazer uma consulta complicada ou utilizar recursos demais